packages/contracts/test/10_unit-testing/12_plugin-setup.ts

```diff
@@ -595,26 +574,70 @@ describe('TokenVotingSetup', function () {
   });
 
   describe('prepareUpdate', async () => {
-    it('should return nothing', async () => {
+    it('update from build 1 should return correct permissions', async () => {
```

Comment:
A good style for test naming is to have a readable sentence, e.g.,
```suggestion
    it('returns the permissions expected for the update from build 1', async () => {
```